tests/test_server.py

```diff
@@ -22,25 +22,39 @@ def test_check_server_running(
     with local_registry:
         assert fdp_serv.check_server_running(LOCAL_REGISTRY_URL)
 
+@pytest.mark.faircli_server
+def test_registry_install_uninstall(mocker: pytest_mock.MockerFixture):
+    with tempfile.TemporaryDirectory() as tempd:
+        reg_dir = os.path.join(tempd, "registry")
+        mocker.patch("fair.common.DEFAULT_REGISTRY_LOCATION", reg_dir)
+        fdp_serv.install_registry(install_dir=reg_dir)
```

Comment:
Maybe there's a fixture that could be used in situations like this to avoid multiple installs.

Reply:
There is a registry fixture but it cannot be used for these commands due to an issue with the virtual environment it uses. To rewrite it would take time.